assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -8794,7 +8794,7 @@ else if (state.getAttribute(ALLPURPOSE_TITLE) != null)
 	private boolean change_from_non_point(SessionState state, String assignmentId, String assignmentContentId, AssignmentContentEdit ac) 
 	{
 		// whether this is an editing which changes non point_grade type to point grade type?
-		if (StringUtils.trimToNull(assignmentId) != null && StringUtils.trimToNull(assignmentContentId) != null)
+		if (StringUtils.trimToNull(assignmentId) != null && StringUtils.trimToNull(assignmentContentId) != null && ac != null)
```

Comment:
Would just use StringUtils.isNotBlank instead of StringUtils.trimToNull(assignmentId) != null

Reply:
Sure